editor/modes/visual-editor/block-list.js

```diff
@@ -248,18 +268,24 @@ class VisualEditorBlockList extends Component {
 						onKeyDown={ noop }
 					/>
 				}
-				<div className="editor-visual-editor__continue-writing">
+				<div
+					className={ continueWritingClassname }
+					onFocus={ this.onContinueWritingFocus }
+					onBlur={ this.onContinueWritingBlur }
+				>
 					<Inserter position="top right" />
 					<button
```

Comment:
We could override the `outline` style in this specific case, or if it's something that would be common, perhaps a prop on `IconButton` (`focusOutline={ false }`).